Libplanet/Tx/Transaction.cs

```diff
@@ -446,7 +451,7 @@ public static Transaction<T> Deserialize(byte[] bytes, bool validate = true)
                     ts,
                     actionsArray
                 ).EvaluateActions(
-                    default(HashDigest<SHA256>),
+                    emptyBlockHash,
```

Comment:
Can't we use just `default(BlockHash)` here?

Reply:
No, since `BlockHash` is unsized, `default(BlockHash)` resembles `new byte[0]` in memory layout, whereas `default(HashDigest<SHA256>)` resembles `new byte[32]`.

Reply:
👌